lib/rational/include/Rational.h

```diff
@@ -4,30 +4,31 @@
 
 class Rational {
 public:
-	Rational();
-	Rational(int numerator, int denominator);
+    Rational();
+    explicit Rational(int numerator, int denominator);
 
-	Rational operator+(const Rational& other);
-	Rational operator-(const Rational& other);
-	Rational operator*(const Rational& other);
-	Rational operator/(const Rational& other);
+    Rational& operator+=(const Rational& other);
+    Rational& operator-=(const Rational& other);
+    Rational& operator*=(const Rational& other);
+    Rational& operator/=(const Rational& other);
 
-	double toDouble();
+    double toDouble();
```

Comment:
Should be const method, the name should start with a capital letter ToDouble, please check others methods in the file and fix first letter